tests/ts_project/src/jsx_close_tag.tsx

```diff
@@ -0,0 +1 @@
+<div
```

Comment:
Beside this you should write simple test to validate response from server [here](https://github.com/pmizio/typescript-tools.nvim/blob/master/tests/requests_spec.lua).

Reply:
Before opening this PR, I didn't even know that there were test case tools available on Neovim. I tried to write a test case(https://github.com/pmizio/typescript-tools.nvim/pull/185/commits/dca9e087d5bac25fd6a1663da1f7fbeb83cd20ac), but the test result did not pass, and I haven't found the specific reason for this.

Reply:
@PHSix checkout my new comments, one of them solve this problem in fact your test is correct you just opened not existing file - `*.ts` instead `*.tsx`

Reply:
yes, thank for you help